README.md

```diff
@@ -1,7 +1,95 @@
 # BestFlow
 We are going to fix bad chatbots!
 
-Hosted at: https://bestflow.netlify.app/
+Diagnostic tool for chatbot transcripts that help retailers identify poor or strong questions and intents through an easily digestible and intuitive display.
 
-This project uses:
-React, Axios, Node, Tailwind, MongoDB, Express and more :D
+We aim to:
+- Reduce time spent on chatbot optimization
+- Reduce bad chatbot intents
+- Display an average of 70% of diagnostic problems in any transcripts’ flow
```

Comment:
```suggestion
- Display an average of 30% of diagnostic problems in any 10 transcripts’ flow
```
I think we decided on 30% of identification across any 10 transcripts. We may wanna workshop this statistic in our next meeting to either confirm that it is correct or a new statistic that's easier to obtain

Reply:
then again, we might not wanna say 10 transcripts in case they think they need to upload 10 transcripts LOL

Reply:
I can put in 30% for now and we can change it later on